api/rs/slint/mcu.md

```diff
@@ -23,21 +23,26 @@ In the following sections we assume that your setup is complete and you have a n
 
 ## Changes to `Cargo.toml`
 
-A typical line in Cargo.toml looks like that:
+Start by adding a dependency to the `slint` crate to your `Cargo.toml`:
 
 ```toml
 [dependencies]
 slint = { version = "0.3.0", default-features = false, features = ["compat-0.3.0", "unsafe-single-threaded", "libm"] }
 # ... other stuff
 ```
 
-Slint uses the standard library by default, so we need to disable the default features.
-Then you need the `compat-0.3.0` feature ([see why in this blog post](https://slint-ui.com/blog/rust-adding-default-cargo-feature.html))
+The default features of the `slint` create are tailored towards hosted environments and includes the "std" feature. In bare metal environments,
+you need to disable the default features and select at least the `compat-0.3.0` feature (see [this blog post](https://slint-ui.com/blog/rust-adding-default-cargo-feature.html)
```

Comment:
why is this feature not a bullet point like the other?

Reply:
Good point! Thanks for spotting, I'll fix that.